### PR TITLE
Rename custom lints to use snake_case

### LINKED ISSUE #65: Update lint rule names to use snake_case instead of kebab-case

### PR DESCRIPTION
**Pull request by solid-vovabeloded**
Closes #65

### CODE REVIEW

File: lib/lints/member_ordering/models/member_ordering_parameters.dart [hunk 1]
```diff
@@ -29,7 +29,7 @@ import 'package:solid_lints/lints/member_ordering/models/member_group/member_gro
 class MemberOrderingParameters {
   /// Config keys
   static const _orderConfig = 'order';
-  static const _widgetsOrderConfig = 'widgets-order';
+  static const _widgetsOrderConfig = 'widgets_order';
   static const _alphabetizeConfig = 'alphabetize';
   static const _alphabetizeByTypeConfig = 'alphabetize-by-type';
```
Comment (yurii-prykhodko-solid) [inline lib/lints/member_ordering/models/member_ordering_parameters.dart:34]: ?
```suggestion
  static const _alphabetizeByTypeConfig = 'alphabetize_by_type';
```

**Review by illia-romanenko — APPROVED** [review]
LGTM

**Review by yurii-prykhodko-solid — CHANGES_REQUESTED** [review]
All LGTM, except one missed spot.

**Review by yurii-prykhodko-solid — APPROVED** [review]
LGTM!

**Pull request merged**